source/query-with-natural-language/enable-natural-language-querying.txt

```diff
@@ -69,25 +68,33 @@ Steps
            - In the top menu bar, click :guilabel:`MongoDB Compass`.
            - ``⌘`` + ``,``
 
-   .. step:: Connect to your Atlas account
+   .. step:: Enable the AI Features Setting and Login to Atlas
+
+      You only need to enabled AI features if it is your first time 
+      using natural language querying.
 
-      a. Select :guilabel:`Feature preview`.
+      a. Select :guilabel:`Artificial Intelligence`.
+      #. Click the :guilabel:`Enable AI Features` check box.
       #. Click :guilabel:`Log in with Atlas`.
 
          A web browser displays the
          `Atlas login page <https://account.mongodb.com/account/login>`_.
 
       #. Complete the login procedure.
 
-         Once completed a :guilabel:`Logged in with Atlas account`
+         Once completed a 
+         :guilabel:`You can create queries and aggregations with 
```

Comment:
[question] Do we need to repeat the message text?

```suggestion
         confirmation 
```

Reply:
It makes more sense IMO to list the message verbatium to confirm the setting is enabled.